modules/appeals_api/app/workers/appeals_api/appeal_received_job.rb

```diff
@@ -38,6 +40,7 @@ def hlr_received
       return Rails.logger.error "AppealReceived: could not find template id for #{template_name}" if template_id.blank?
 
       vanotify_service.send_email(params({ template_id: template_id }))
+      StatsD.increment(STATSD_CLAIMANT_EMAIL_SENT, tags: { appeal_type: 'hlr' })
```

Comment:
We have two types of emails we send out - the veteran email and the non-veteran-claimant email. I think we should tag which one we used in each of these increments. The `claimant?` method should be easy to ternary into the right value.

Reply:
done